pkg/controllers/job/plugins/ssh/ssh_test.go

```diff
@@ -68,6 +103,23 @@ func TestSSHPlugin(t *testing.T) {
 			if plugin.sshPublicKey != test.sshPublicKey {
 				t.Errorf("Expected sshPublicKey=%s, got %s", test.sshPublicKey, plugin.sshPublicKey)
 			}
+
+			gotErr := plugin.OnPodCreate(pod, job)
+			if !reflect.DeepEqual(gotErr, test.wantErr) {
+				t.Errorf("OnPodCreate error = %v, wantErr %v", gotErr, test.wantErr)
```

Comment:
Maybe we  should stop from continuing the left case. Other cases have same issue.

Reply:
Sorry, I don't understand what it means.

Reply:
Does it mean to remove the OnPodCreate method?

Reply:
@lowang-bh

Reply:
fixed it, use `t.Skip()` to skip when `err != nil`